WEB/Src/DependencyCollector/DependencyCollector/Implementation/AzureSdk/AzureSdkDiagnosticListenerSubscriber.cs

```diff
@@ -2,19 +2,29 @@
 {
     using System;
     using System.Diagnostics;
+    using System.Diagnostics.Tracing;
     using Microsoft.ApplicationInsights.Common;
     using Microsoft.ApplicationInsights.Extensibility;
     using Microsoft.ApplicationInsights.Extensibility.Implementation;
 
-    internal sealed class AzureSdkDiagnosticListenerSubscriber : DiagnosticSourceListenerBase<object>
+    internal sealed class AzureSdkDiagnosticListenerSubscriber : DiagnosticSourceListenerBase<object>, IDisposable
     {
         public const string DiagnosticListenerName = "Azure.";
+        private readonly IDisposable logsListener;
 
         public AzureSdkDiagnosticListenerSubscriber(TelemetryConfiguration configuration) : base(configuration)
         {
+            // listen to Cosmos EventSource only - other logs can be sent using ILogger
+            this.logsListener = new AzureSdkEventListener(this.Client, EventLevel.Informational, "Azure.Cosmos");
```

Comment:
Are these logs fired very rarely? or they can be verbose? Since this is going to be enabled by default, we'll need ensure it wont be too noisy.

Reply:
just saw that PR description answers this qn already..